ImprovedAnvil/lib/spells/00_copy_vanilla.tph

```diff
@@ -339,7 +339,7 @@ COPY ~ImprovedAnvil\resources\spells\SPPR708.spl~ ~override/SPPR708.spl~
 
 COPY ~ImprovedAnvil\resources\spells\SPPR710.spl~ ~override/SPPR710.spl~
   SAY NAME1 @9097
-  SAY UNIDENTIFIED_DESC @9570
+  SAY UNIDENTIFIED_DESC @12082
```

Comment:
@9570 still used somewhere else?

Reply:
probably not, though I don't want to remove 12082, add its content to @9570 and decrease all new string references by 1...

Reply:
@9570 is still used in at least on other place, so it makes sense to check and see if it could be used in both places.
As for removing @12082, you could just leave it as an empty string and use later for something else. There's no forced requirement to remove and decrement all subsequent numbers.

Reply:
Ahh, now I know why I decided to add another string. There are 2 versions of this spell now, Wizards have to touch the opponent, while Priests have a small range, but they still can cast it from afar.

Reply:
Does it make sense to have them be different between priests and wizards, though?

Reply:
To me it does. Wizards can get up to -5 casting speed and Improved Alacrity, so they are far better at casting this spell. Destroying undead should be more of a Priest thing, so to make that even, I thought that they could get a ranged variant, while Wizards have to move close and and risk being hit.